<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineOfSightWidget</class>
 <widget class="QWidget" name="LineOfSightWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Line of Sight Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>16</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLineEdit" name="le_souceLon">
       <property name="placeholderText">
        <string>lon</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="le_targetLat">
       <property name="placeholderText">
        <string>lat</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_sourceLat">
       <property name="placeholderText">
        <string>lat</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Target:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="le_targetLon">
       <property name="placeholderText">
        <string>lon</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Sourece:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="le_sourceAlt">
       <property name="placeholderText">
        <string>alt</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pb_sourceLocation">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="le_targetAlt">
       <property name="placeholderText">
        <string>alt</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pb_targetLocation">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="le_visibleColor">
     <property name="title">
      <string>LOS Line Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblWidth">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblWidth_3">
        <property name="text">
         <string>Style:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cb_lineStyle">
        <item>
         <property name="text">
          <string>Stright Line</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dotted Line</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dashed Line</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="sb_lineWidth">
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pb_visbleColor">
        <property name="text">
         <string>Good Color</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="pb_invisibleColor">
        <property name="text">
         <string>Bad Color</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Result</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="lblWidth_2">
          <property name="text">
           <string>Hit Point:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="le_resultLon"/>
          </item>
          <item>
           <widget class="QLineEdit" name="le_resultLat"/>
          </item>
          <item>
           <widget class="QLineEdit" name="le_resultAlt"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnUpdate">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pb_close">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 4);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
